src/prototype_room_basebuilder.js

```diff
@@ -259,7 +259,7 @@ Room.prototype.setupStructure = function(structure) {
   }
 
   var max = CONTROLLER_STRUCTURES[structure][this.controller.level];
-  for (let pos of (this.memory.position.structure[structure] || [])) {
+  for (let pos of(this.memory.position.structure[structure] || [])) {
```

Comment:
`jsbeautifier` makes this changed based on the settings in the repository. So in the end it is 'fine' (also couldn't find a quick way to change the configuration according :-))